src/current/v25.4/cockroach-debug-zip.md

```diff
@@ -197,6 +197,29 @@ $ cockroach debug zip ./cockroach-data/logs/debug.zip --redact --insecure --host
 server/server.go:1423 ⋮ password of user ‹×› was set to ‹×›
 ~~~
 
+#### Cluster settings redaction
+
+Example of a cluster setting in `crdb_internal.cluster_settings.txt` without redaction enabled:
+
+~~~
+variable	value	type	public	sensitive	reportable	description	default_value	origin
+...
+cluster.organization	Cockroach Labs Testing	s	t	f	f	organization name		override
```

Comment:
nit: its kinda hard to read this table in the rendered markdown. Is it possible to make the columns and headers align better?